kuma/javascript/src/active-banner.jsx

```diff
@@ -189,9 +189,28 @@ function Banner(props: BannerProps) {
     );
 }
 
+export const MDN_BROWSER_COMPAT_REPORT_ID = 'mdn_browser_compat_report';
 export const DEVELOPER_NEEDS_ID = 'developer_needs';
 export const SUBSCRIPTION_ID = 'subscription_banner';
 
+function MDNBrowserCompatReportBanner() {
+    return (
+        <Banner
+            id={MDN_BROWSER_COMPAT_REPORT_ID}
+            className="developer-needs"
```

Comment:
```suggestion
            classname="developer-needs"
```

Oops. I didn't realize this hack uses that typical React prop name but in lowercase. Ugly. :(